scope.cpp

```diff
@@ -110,6 +110,8 @@ Scope::Scope()
     val = pConfig->Profile.GetBoolean(prefix + "/UseDecComp", true);
     EnableDecCompensation(val);
 
+    m_hasHPEncoders = pConfig->Profile.GetBoolean("/scope/HiResEncoders", false);

```

Comment:
do we need to also set m_hasHPEncoders when the GA is run? it looks like as it is now the GA would update the value in pConfig, but the scope's m_hasHPEncoders would not get updated until the scope object is next instantiated (typically next time the user starts phd2)